src/devices/Adxl357/README.md

```diff
@@ -0,0 +1,31 @@
+# ADXL357 - Accelerometer
+ADXL357 is a 3-Axis digital accelerometer 40g with 20-bit resolution measurement at up to ±40g.
```

Comment:
Markdown 101: you have to have a new line after a # or any other title (so apply to the rest of the file)

Reply:
I added new lines after every title.